main.go

```diff
@@ -207,7 +405,7 @@ func main() {
 				Message:   fmt.Sprintf("lookup type %T not known", x),
 			}
 			log.Error(&svcerr)
-			if err = conn.Publish(reply, &svcerr); err != nil {
+			if err = conn.Publish(msg.Reply, &svcerr); err != nil {
```

Comment:
Obviously (I think) we'd eventually want these to pass in a context (and thus a span, which could get injected into the headers). Looks like you added `PublishContext` above to handle that, so I'm assuming that's just a todo still

Reply:
This should be fixed now.